misc-utils/lsfd-unkn.c

```diff
@@ -23,7 +23,9 @@
 #include <sys/syscall.h>
 #include <sys/timerfd.h>
 #include <time.h>
+#include <unistd.h>
```

Comment:
This is for the call to `syscall()` in `pidfd-utils.h`, correct?
If yes, please add a preparatory commit adding the include directly to `pidfd-utils.h`.

Reply:
Will do.